FEATURE_MATRIX.md

```diff
@@ -38,21 +45,21 @@
 ## Planning
 | | C# | Python | Notes |
 |---|---|---|---|
-| Plan | ✅ | ❌ | |
-| SequentialPlanner | ✅ | ❌ | |
+| Plan                              | ✅ | ❌ | |
+| SequentialPlanner                 | ✅ | ❌ | |
 
 ## Connectors and Skill Libraries
 | | C# | Python | Notes |
 |---|---|---|---|
 | Qdrant (Memory)                   | ✅ | ❌ | Vector optimized |
-| ChromaDb (Memory)                 | ❌ | 🔄 | |
+| ChromaDb (Memory)                 | ❌ | ✅ | |
```

Comment:
Is ChromaDB Vector Optimized? Would it be better to highlight which are not vector optimized?

Reply:
Chroma is an embedding db, so I believe it must be vector optimized. Per your suggestion, I've removed the "Vector optimized" on all of the entries, only calling out the ones that are not.